style.css

```diff
@@ -191,6 +191,83 @@ a {
   width: 18px;
 }
 
+.card-container {
+  position: fixed;
```

Comment:
- [ ] Your mobile popup window is having overlaying and positioning issues. Make the main project page behind the popup window to have a fixed position such that it does not scroll when the popup displays.
- [ ] Check the screenshots below and fix your mobile display to look same like the figma design
- Your design
![image](https://user-images.githubusercontent.com/62849872/210932062-77fd791c-be27-44c7-9432-2d8191e79189.png)
Looking at your design one has to scroll to the right to read the entire project description and see the close button
![image](https://user-images.githubusercontent.com/62849872/210932667-ce4a3bcf-b40d-4b72-8fe1-f60006eea404.png)

- Figma design
![image](https://user-images.githubusercontent.com/62849872/210932187-4c033fd8-ad6f-4971-aa00-a58c272214ab.png)